android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -128,6 +138,37 @@ public static void setNextRewardsOnboardingModalDate(long nextDate) {
         sharedPreferencesEditor.apply();
     }
 
+    public static long getRewardsOnboardingIconTiming() {
+        return ChromeSharedPreferences.getInstance().readLong(PREF_REWARDS_ONBOARDING_ICON_TIMING);
+    }
+
+    public static void setRewardsOnboardingIconTiming(long nextDate) {
+        ChromeSharedPreferences.getInstance().writeLong(
+                PREF_REWARDS_ONBOARDING_ICON_TIMING, nextDate);
+    }
+
+    public static long getRewardsOnboardingIconInvisibleTiming() {
+        return ChromeSharedPreferences.getInstance().readLong(
+                PREF_REWARDS_ONBOARDING_ICON_INVISIBLE_TIMING);
+    }
+
+    public static void setRewardsOnboardingIconInvisibleTiming(long nextDate) {
+        ChromeSharedPreferences.getInstance().writeLong(
+                PREF_REWARDS_ONBOARDING_ICON_INVISIBLE_TIMING, nextDate);
+    }
+
+    public static void setRewardsOnboardingIconInvisible(boolean hasUpdated) {
+        SharedPreferences.Editor sharedPreferencesEditor =
+                ContextUtils.getAppSharedPreferences().edit();
+        sharedPreferencesEditor.putBoolean(PREF_REWARDS_ONBOARDING_ICON_INVISIBLE, hasUpdated);
+        sharedPreferencesEditor.apply();
+    }
+
+    public static boolean hasRewardsOnboardingIconInvisibleUpdated() {
+        return ContextUtils.getAppSharedPreferences().getBoolean(
+                PREF_REWARDS_ONBOARDING_ICON_INVISIBLE, false);
```

Comment:
`android/java/org/chromium/chrome/browser/BraveRewardsHelper.java:168: Use SharedPreferencesManager instead to access app-wide SharedPreferences from //src/chrome.`